setup.py

```diff
@@ -36,10 +36,10 @@
 import vsc.install.shared_setup as shared_setup
 from vsc.install.shared_setup import ag, kh, jt, sdw
 
-VSC_INSTALL_REQ_VERSION = '0.9.19'
+VSC_INSTALL_REQ_VERSION = '0.10'
 
 PACKAGE = {
-    'version': '2.4.18',
+    'version': '2.5',
```

Comment:
well, seems like you didn't for the last vsc-install, but there it was impossible to see...

I think we're pretty much sticking to semver otherwise?